core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -310,6 +310,9 @@ public OutputStream create(String path) throws IOException {
    * @throws IOException if a non-Alluxio error occurs
    */
   public OutputStream create(String path, CreateOptions options) throws IOException {
+    if (!options.isEnsureAtomic()) {
```

Comment:
@aaudiber That might be difficult across UFSs as object stores do not have direct writes. !ensureAtomic actually does not ensure anything. I suggest we revisit this when the journal API is designed.